Assignment3/Assignment3.py

```diff
@@ -21,6 +21,7 @@ def main():
     sql1 = "select * from baseball.batter_counts"
     database = "baseball"
     user = "bita"
+    password = "salam"
```

Comment:
Bita don't add your your password onto a file on github, you want to use an environment variable or something so people dont know it.

Reply:
Done

Reply:
If you added the pre-commit hook, it would have caught that and not let it get committed!